angularSTEP/src/app/home/home.component.scss

```diff
@@ -0,0 +1,18 @@
+.example-container{
+    text-align: center;
+}
+
+.form-container{
+    width: 400px; 
+    margin: 0 px auto; 
```

Comment:
Here's at least one remaining instance of a measured value containing a space between the number and unit. Note that for 0 values, the unit is unnecessary and usually omitted since it makes no difference in the outcome

Reply:
Good eye! I thought I fixed all of those little issues, thanks